canonical/ubuntu.com
# CVEs landing page redesign

## Done

- Added redesign as per [mock-up](https://www.figma.com/file/nUJEBpOPTW74RqDRuMQXxB/23.04-U.com---CVEs?node-id=949%3A22802&mode=dev)

## Future work

The releases links will be handled in a separate PR, as will the advanced search link.

## QA

- View the site locally in your web browser at: https://ubuntu-com-13382.demos.haus/security/cves
    - Be sure to test on mobile, tablet and desktop screen sizes
- See that the page matches the design

## Issue / Card

Fixes https://warthogs.atlassian.net/browse/WD-5145
